llvm/unittests/ProfileData/MemProfTest.cpp

```diff
@@ -18,14 +18,23 @@
 #include "llvm/ProfileData/MemProfData.inc"
 #include "llvm/ProfileData/MemProfRadixTree.h"
 #include "llvm/ProfileData/MemProfReader.h"
+#include "llvm/ProfileData/MemProfSummary.h"
 #include "llvm/Support/raw_ostream.h"
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
 #include <initializer_list>
 
+using namespace llvm;
+
+extern cl::opt<float> MemProfLifetimeAccessDensityColdThreshold;
```

Comment:
Can we move these extern decls inside the llvm namespace below instead of adding a broad using directive above?

Reply:
This turned out to not work, because the variables themselves are not defined in the llvm namespace. I was able to replace the using namespace with just adding "llvm::" qualifiers in front of the cl::opt types.